website/docusaurus.config.js

```diff
@@ -309,6 +309,27 @@ const config = {
         ],
       },
     ],
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'api',
+        path: 'api',
+        routeBasePath: 'api',
+      },
+    ],
+    [
+      'docusaurus-plugin-typedoc',
+      {
+        id: 'api',
+        plugin: ['typedoc-plugin-markdown'],
+        entryPoints: ['..\\packages\\extension-api\\src\\extension-api.d.ts'],
```

Comment:
FYI the path is not working on macOS (it says path not found)

if slashes are not working on windows, probably uses javascript with ${path.sep}

Reply:
I replaced with `require.resolve`

Reply:
require is kind of forbidden these days

could you add

import {resolve} from 'node:path';

and then use resolve('') (without require prefix)